src/main/services/agreementsService/api.ts

```diff
@@ -113,12 +114,24 @@ const getAgreementLotEventTypes = async (agreementId: string, lotId: string): Pr
   );
 };
 
+// GET /agreements/:agreement-id
+const getAgreementsServiceHealth = async (): Promise<FetchResult<object>> => {
+  return genericFecthGet<object>(
```

Comment:
I know you are working on adding the health endpoint to the Agreement Service, once you know what it looks like you should create a Type for it (in the types file)

Reply:
So I actually have this in place already since I can create the type if that's how we want to go.  But do we need this?  I did think about it, but decided since all we're interested in is the 200 response using an object would be fine.  Happy to change if you feel differently though!